model_fitting/random_forest.py

```diff
@@ -188,40 +189,37 @@ def train_models(csv_file_path, done_path, logs_dir,error_path, num_of_configura
         os.makedirs(output_path_i, exist_ok=True)
         file_save_configuration = save_configuration_to_txt_file(configuration, output_path_i)
         logging.info(f'Configuration #{i} hyper-parameters are:\n{configuration}')
-        done_file_path=os.path.join(logs_dir, f'{model_number}_done_train_random_forest.txt')
-        cmds=[file_save_configuration, csv_file_path, is_hits_data, output_path_i,
+        done_file_path=os.path.join(logs_dir, f'{model_number}_{csv_file_prefix}_done_train_random_forest.txt')
+        cmds=[file_save_configuration, csv_file_path, rank_method, output_path_i,
                 model_number, done_file_path, '--random_forest_seed', random_forest_seed,
                 '--cv_num_of_splits', cv_num_of_splits]
         if not os.path.exists(done_file_path):
             all_cmds_params.append(cmds)
         else: 
             logger.debug(f'Skipping random forest train as {done_file_path} found')
-           
-    
-    queue_name = 'pupkoweb'
-    verbose = True
-    executable = 'python'

```

Comment:
Don't remove, we want to be backward compatible

Reply:
I returned the variable executable, the other I bring the variable from module wraper.